package.json

```diff
@@ -26,7 +26,10 @@
     "url": "https://github.com/JBaczuk/bscript-parser/issues"
   },
   "homepage": "https://github.com/JBaczuk/bscript-parser#readme",
-  "dependencies": {},
+  "dependencies": {
+    "bech32": "^1.1.3",
```

Comment:
lock these dependencies

Reply:
@JBaczuk This is done.  I also rebased to resolve the conflict in the package.json file.